src/main/java/org/skriptlang/skript/lang/converter/Converters.java

```diff
@@ -304,8 +304,13 @@ private static <F, T extends ParentType, SubType extends F, ParentType> Converte
 
 		// Attempt to find converters that have either 'from' OR 'to' not exactly matching
 		for (ConverterInfo<?, ?> unknownInfo : CONVERTERS) {
+			int flags = unknownInfo.getFlags();
 			if (unknownInfo.getFrom().isAssignableFrom(fromType) && unknownInfo.getTo().isAssignableFrom(toType)) {
 				ConverterInfo<F, ParentType> info = (ConverterInfo<F, ParentType>) unknownInfo;
+				if ((flags & Converter.ALLOW_UNSAFE_CASTS) == 0) {
+					if ((flags & Converter.NO_RIGHT_CHAINING) == Converter.NO_RIGHT_CHAINING
+						&& !toType.isAssignableFrom(unknownInfo.getTo())) continue;
```

Comment:
```suggestion
						&& !toType.isAssignableFrom(unknownInfo.getTo()))
						continue;
```
`continue` should be on the next line for each